### PR TITLE
add countdown

### PR DESCRIPTION
**Pull request by prineth**

### CODE REVIEW

File: src/components/Countdown/CountdownTimer.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+};
+
+const CountdownTimer = ({ targetDate }) => {
+  const [days, hours, minutes, seconds] = useCountdown(targetDate);
+
+  return (
+    <>
+      <div className="bg-hero-pattern grid grid-cols-1 grid-rows-6 h-85 place-items-center">
+        <p className="font-bold text-12xl text-brown-10 leading-15 mt-11">
+          Lorem Ipsum
```
Comment (SanjulaD) [inline src/components/Countdown/CountdownTimer.js:42]: check shared component for adding titles

File: src/components/Countdown/countdownHooks/useCountdown.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      setCountDown(countDownDate - new Date().getTime());
+    }, 1000);
+
+    return () => clearInterval(interval);
+  }, [countDownDate]);
+
+  return getReturnValues(countDown);
+};
+
+const getReturnValues = (countDown) => {
```
Comment (SanjulaD) [inline src/components/Countdown/countdownHooks/useCountdown.js]: we can use separate file for these calculations. eg: utils.js keep it main folder

File: src/pages/index.js [hunk 3]
```diff
@@ -1,8 +1,14 @@
 import Head from "next/head";
+import CountdownTimer from "../components/Countdown/CountdownTimer";
 import { SpeakerSection } from "../components/Speakers/SpeakerSection";
 import { TrackSession } from "../components/Tracks/TrackSession";
 
 export default function Home() {
+  const THREE_DAYS_IN_MS = 3 * 24 * 60 * 60 * 1000;
```
Comment (SanjulaD) [inline src/pages/index.js]: use seprate file for calculations

**Review by SanjulaD — CHANGES_REQUESTED** [review]
change requested

**Comment by SanjulaD** [comment]
@prineth 
image

File: src/components/Countdown/CountdownTimer.js (hunk 1 above)
Comment (prineth) [reply src/components/Countdown/CountdownTimer.js:42]: It has unnecessary margin-top. That's why I didn't use it.

File: src/components/Countdown/countdownHooks/useCountdown.js (hunk 2 above)
Comment (prineth) [reply src/components/Countdown/countdownHooks/useCountdown.js]: Sure. I'll do it.

File: src/pages/index.js (hunk 3 above)
Comment (prineth) [reply src/pages/index.js]: Sure. I'll do it.

File: src/components/Countdown/CountdownTimer.js (hunk 1 above)
Comment (SanjulaD) [reply src/components/Countdown/CountdownTimer.js:42]: check removing or make it high order

File: src/pages/index.js (hunk 3 above)
Comment (prineth) [reply src/pages/index.js]: done

File: src/components/Countdown/countdownHooks/useCountdown.js (hunk 2 above)
Comment (prineth) [reply src/components/Countdown/countdownHooks/useCountdown.js]: done

**Review by SanjulaD — APPROVED** [review]

**Pull request merged**